### ISSUE TITLE
Test fails when cloning a new copy and reinstalling the dependencies

### ISSUE THREAD

**Issue by FSMaxB** [josdejong/mathjs#477]
When cloning a new copy of mathjs and installing the dependencies via `npm install`. Suddenly a test fails that didn't fail before. This happens on `master` and `develop`.

```
  3484 passing (3s)
  19 pending
  1 failing

  1) mode should not throw an error if the input contains mixture of array and non-array values:
     TypeError: Too many arguments in function mode (expected: 1, actual: 3)
      at createError (node_modules/typed-function/typed-function.js:135:17)
      at mode (eval at _typed (node_modules/typed-function/typed-function.js:1039:22), <anonymous>:15:13)
      at Context.<anonymous> (test/function/statistics/mode.test.js:37:22)
```

**Comment by FSMaxB** [comment]
Could this be a bug in typed-function?

**Comment by josdejong** [comment]
This failing unit test is indeed caused by `typed-function` v0.10.2 which I released today. 

Until this is fixed, we can work around by changing the `typed-function` dependency in package.json to `"0.10.1"` (instead of `"^0.10.1"`). The unit test itself is testing quite an edge case, so people installing mathjs right now will most likely not suffer from this. I will fix it asap, hopefully tomorrow.

**Comment by FSMaxB** [comment]
Interesting. I had made the assumption that `^` in package.json had the same meaning as `^` in regular expressions. I didn't know it means "this version and up", like an arrow pointing upwards.

**Comment by josdejong** [comment]
I've just released `typed-function` v0.10.3 where I reverted the fix of v0.10.2 which gave this issue. See also https://github.com/josdejong/typed-function/pull/8. The original issue still needs to be fixed (and was not causing issues for math.js), but now we can take the time for it.
